app/controllers/katello/api/v2/api_controller.rb

```diff
@@ -94,6 +97,12 @@ def scoped_search(query, default_sort_by, default_sort_order, resource = resourc
 
       query = query.order("#{sort_attr} #{params[:sort_order] || default_sort_order}")
 
+      if includes.is_a?(Proc)
```

Comment:
well i guess you can accomplish that without a proc actually.  I will remove and we can always add back.